app/src/utils/enums/application.ts

```diff
@@ -30,6 +30,11 @@ export enum Initiative {
   HOUSING = 'HOUSING'
 }
 
+export enum UserStatus {
+  APPROVED = 'Approved',
+  PENDING = 'Pending',
+  REJECTED = 'Rejected'
+}
 export enum Regex {
```

Comment:
Style: Please ensure new lines are between blocks. I've noticed this a few times within this PR.

Reply:
done